ui/page.go

```diff
@@ -512,8 +514,26 @@ func (c Container) Layout(gtx layout.Context, w layout.Widget) layout.Dimensions
 	return c.padding.Layout(gtx, w)
 }
 
+var (
+	MaxWidth = unit.Dp(800)
+)
+
 func (page pageCommon) UniformPadding(gtx layout.Context, body layout.Widget) layout.Dimensions {
-	return layout.UniformInset(values.MarginPadding24).Layout(gtx, body)
+	width := gtx.Constraints.Max.X
+
+	padding := values.MarginPadding24
+
+	if (width - 2*gtx.Px(padding)) > gtx.Px(MaxWidth) {
+		paddingValue := float32(width-gtx.Px(MaxWidth)) / 2
+		padding = unit.Px(paddingValue)
```

Comment:
This can be a single line
```suggestion
		padding = unit.Px(float32(width-gtx.Px(MaxWidth)) / 2)
```